### PR TITLE
Fix spacing and layout of registered TopBar items

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

Before:
Screen Shot 2022-03-10 at 2 54 48 PM

After (with window buttons):
Screen Shot 2022-03-10 at 2 51 10 PM

After (without window buttons):
Screen Shot 2022-03-10 at 2 51 37 PM

### CODE REVIEW

File: src/renderer/components/layout/top-bar/top-bar.tsx [hunk 1]
```diff
@@ -100,28 +97,24 @@ const NonInjectedTopBar = observer(({ items, children, isWindows, isLinux, ...re
...
           data-testid="history-forward"
           material="arrow_forward"
-          className="ml-5"
           onClick={goForward}
           disabled={!nextEnabled.get()}
         />
       </div>
-      <div className={styles.controls}>
+      <div className={styles.items}>
         {renderRegisteredItems(items.get())}
```
Comment (Nokel81) [inline src/renderer/components/layout/top-bar/top-bar.tsx:117]: Note, it should now be relatively easy to add the ability to add items to either side via the extension API. That, however, is left for future work.

**Comment by stefcameron** [comment]
Thank you for the fix!

**Review by aleksfront — APPROVED** [review]
LGTM

**Review by jim-docker — COMMENTED** [review]
Seems to have a default hover (hovering over the middle icon):

Screen Shot 2022-03-14 at 3 12 46 PM

**Comment by Nokel81** [comment]
Would you mind providing the extension registration code?

**Comment by jim-docker** [comment]
> Would you mind providing the extension registration code?

ah, never mind, just realized it is coming from my extension, sorry

**Review by jim-docker — APPROVED** [review]
I guess it's pretty much designer beware:

Screen Shot 2022-03-14 at 3 41 26 PM

I guess we don't clip to the top bar so we can purposefully let stuff bleed outside

**Comment by Nokel81** [comment]
Yeah we don't clip it at all.

Can you merge this PR once the release has been made?

**Comment by jim-docker** [comment]
> Yeah we don't clip it at all.
> 
> Can you merge this PR once the release has been made?

do we need to wait?

**Comment by Nokel81** [comment]
No, I realized that that release is from a different branch.

**Pull request merged**